release-notes/kvm/v11.1.1.md

```diff
@@ -0,0 +1,5 @@
+## :zap: Giant Swarm Release 11.1.1 for KVM is now active for you! :zap:
+
+### nginx-ingress-controller v0.28.0 ([Giant Swarm App v1.3.0](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#v130-2020-01-30))
+
+- Updated from upstream `ingress-nginx` v0.26.1 - [changelog](https://github.com/kubernetes/ingress-nginx/releases/tag/nginx-0.28.0).
```

Comment:
@cokiengchiara we likely need to communicate breaking change with default SSL ciphers https://github.com/giantswarm/giantswarm/issues/8851#issuecomment-583723774